examples/connectivity/plot_adhd_tangent.py

```diff
@@ -0,0 +1,107 @@
+"""
+Comparing different connectivity measures
+=========================================
+
+This example compares different measures of functional connectivity between
+regions of interest.
+
+"""
+
+# Fetch dataset
+import nilearn.datasets
+atlas = nilearn.datasets.fetch_atlas_msdl()
+dataset = nilearn.datasets.fetch_adhd()
+
+# Extract and preprocess regions time series
+import nilearn.input_data
+import joblib
```

Comment:
you should import joblib from sklearn.externals. There is no guarantee that users have joblib installed.